### PR TITLE
Settings page

### PR DESCRIPTION
**Pull request by milkovsky**

### CODE REVIEW

File: src/Form/RulesSettingsForm.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      '#default_value' => $config->get('debug_log'),
+    );
+
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
```
Comment (M1r1k) [inline src/Form/RulesSettingsForm.php:67]: :D Where did you find this form? Seems like copy/paste from user module :)

**Comment by M1r1k** [comment]
Also add WebTest test coverage

File: src/Form/RulesSettingsForm.php (hunk 1 above)
Comment (milkovsky) [inline src/Form/RulesSettingsForm.php:67]: facepalm

**Comment by klausi** [comment]
Now that we have rules_ui we should move this there.

File: src/Tests/RulesSettingsTest.php [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+use Drupal\Component\Utility\Html;
+use Drupal\simpletest\WebTestBase;
+
+/**
+ * Tests for rules module configuration.
+ *
+ * @group rules
+ */
+class RulesSettingsTest extends WebTestBase {
```
Comment (milkovsky) [inline src/Tests/RulesSettingsTest.php:18]: Needs work, only started.

**Comment by dasjo** [comment]
https://github.com/fago/rules/pull/244

**Pull request closed without merging**